### ISSUE TITLE
Comments on FedRAMP Tailored from: info@fedramp.gov

### ISSUE THREAD

**Issue by rhoesing** [GSA/fedramp-tailored#40]
Ashley & Ed:
Attached are a couple of comments on the Tailored FedRAMP document. I just had a few and figured I would share with both of you since we were at the Cloud CCOE meeting together and this came up as a discussion point.

Thanks,
Mike Cassidy

[-Mike Cassidy FedRAMP Tailored Comments .pdf](https://github.com/GSA/fedramp-tailored/files/828984/-Mike.Cassidy.FedRAMP.Tailored.Comments.pdf)

**Comment by konklone** [comment]
For those following by email, the two comments here are:

* In reference to Tailored referencing NIST's cloud definition: "For example YouTube, Facebook, etc., are not Cloud services, they are in my mind social media websites that do not meet but one or two of the Cloud essential elements."

* In reference to the use of the term `Low-Impact Software-as-a-Service (LISaaS)`: " Minor Comment – Do we want to add another XaaS acronoym? I would recommend using LI SaaS and separate into two words."

**Comment by FedRAMP** [comment]
Thank you for your suggestions. We now refer readers to the NIST 800-145 definition of “Software as a Service (SaaS),” which we believe more accurately reflects our intentions. We also adopted "LI-SaaS" as the standard for referring to Low Impact SaaS systems. We did not add the questions about geography and citizenship as some agencies do not have these restrictions. We have instead enhanced the attestation appendix to include the collection of this information.
